packages/flutter_tools/lib/src/cache.dart

```diff
@@ -385,7 +385,7 @@ class FlutterWebSdk extends CachedArtifact {
     } else if (platform.isLinux) {
       platformName += 'linux-x64';
     } else if (platform.isWindows) {
-      platformName = 'windows-x64';
+      platformName += 'windows-x64';
```

Comment:
I wanted to suggest using a `StringBuffer` then thought it would be too much of a nitpick. In this case though, it would have prevented this error :)